recipes/metplus/meta.yaml

```diff
@@ -0,0 +1,125 @@
+{% set version = "6.0.0" %}
+{% set met_version = "12.0.2" %}
+
+package:
+  name: metplus
+  version: {{ version }}
+
+source:
+  - url: https://github.com/dtcenter/METplus/archive/refs/tags/v{{ version }}.tar.gz
+    md5: a5612dfd025055707f4a45dff028e628
+    sha256: e9358aede2fd2abecd81806227de7b165d68fdf2fc9defcbba24df229461b155
+
+  - url: https://github.com/dtcenter/MET/archive/refs/tags/v{{ met_version }}.tar.gz
+    md5: 4b9dba80639a74f1a3ecb15357785603
+    sha256: 613c74cbe5a0afa87422e81a4cd70984238c1a0eaf1a395503486331223ede45
+    folder: MET
+
+  - url: https://downloads.sourceforge.net/project/gs-fonts/gs-fonts/8.11%20%28base%2035%2C%20GPL%29/ghostscript-fonts-std-8.11.tar.gz
+    md5: 6865682b095f8c4500c54b285ff05ef6
+    sha256: 0eb6f356119f2e49b2563210852e17f57f9dcc5755f350a69a46a0d641a0c401
+    folder: gs-fonts
+
+
+build:
+  skip: True  # [win]
+  skip: True  # [py!=310]
+  number: 0
+  entry_points:
+    - run_metplus.py = metplus.scripts.run_metplus:cli_main
+
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ stdlib("c") }}
+    - {{ compiler('fortran') }}
+    - {{ compiler('cxx') }}
+    - cmake
+    - make
+    - llvm-openmp  # [osx]
+    - libgomp      # [linux]
+    - gnuconfig
+
+  host:
+    - python
+    - pip
+    - gsl
+    - netcdf-cxx4
+    - zlib
+    - cairo
+    - freetype
+    - nceplibs-bufr
+    - nceplibs-g2c
+    - jasper
+    - proj
```

Comment:
These libraries are providing unused run_exports. Are they python libraries like numpy? Else why are the dependencies if they aren't used at runtime?

https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=1227610&view=logs&j=4cabe686-70ae-553a-7fd0-310379f2cbac&t=6a4fc7c9-c31a-5115-eff9-6479d72b69ff&l=13316

Reply:
@carterbox, jasper and proj are dependencies of the MET C++ apps. However, I ran a test build on my machine without these listed as host requirements and it appears to work. Perhaps these are made available through another dependency and therefore don't need to be explicitly included here? I can remove them if that is the case.

numpy is a python library that is required to run certain functionality in the MET apps. Should I include numpy under ignore_run_exports?

Also, I see libopenblas throws this warning for MacOS. I believe this was added to support one of the dependencies that I previously installed in the recipe but now pull from conda-forge, so I can remove that as well.

Reply:
The host requirements section is for dependencies that are needed at compile/build time for linking purposes. If this program isn't using header files from jasper and proj and linking to binaries from those packages, it shouldn't be needed in the host requirements. The same goes for numpy. If you aren't using the numpy C-API, then it doesn't need to be listed in the host requirments.

Reply:
Here's our documentation about packages that link to the numpy C-API.

https://conda-forge.org/docs/maintainer/knowledge_base/#building-against-numpy